source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,518 @@
+
+The Edge Server Service automatically infers the other fields from the shape 
+of the documents you sync or the documents in the collection.
+
+For information about how to add a schema, refer to :ref:`enforce-a-schema`.
+
+Change Fields in a Document
+```````````````````````````
+
+When you change the fields in a document using an Edge Server Service,
+the service automatically changes the schema to add new fields.
+
+Some changes that you can make through the UI are considered "breaking" 
+schema changes. When a "breaking" schema change occurs, any Edge Server 
+instance with the old schema reboots and wipes itself. The wipe process 
+erases any unsynced data.
+
+Edge Server will get the ability to recover local unsynced data in a future 
+release.
+
+For details about which types of changes are considered "breaking" changes,
+refer to :ref:`breaking-change-quick-reference`.
+
+Device Sync Service
+~~~~~~~~~~~~~~~~~~~
+
+An Edge Server running in a Device Sync Service uses Device Sync rules for
+updating a data model.
+
+When a "breaking" schema change occurs, any Edge Server instance with the old 
+schema reboots and wipes itself. The wipe process erases any unsynced data. 
+
+If that data was written by an Atlas Device SDK client, the client itself
+can handle data recovery. This enables local unsynced data written by the 
+client to eventually make its way to Atlas.
+
+If the data was written by a MongoDB Driver or tool using the Wire Protocol,
+the unsynced data is unrecoverable.
+
+Edge Server will get the ability to recover local unsynced data in a future 
+release.
+
+For more information, refer to :ref:`synced-schema-overview`.
+
+.. _troubleshoot-edge-server-instance:
+
+Troubleshoot an Edge Server Instance
+------------------------------------
+
+A few conditions can cause an Edge Server instance to enter a ``Failed`` state:
+
+- :ref:`edge-server-exceeds-client-maximum-offline-time`
+- :ref:`edge-server-more-than-one-major-version-behind`
+- :ref:`edge-server-token-invalid`
+
+.. _edge-server-exceeds-client-maximum-offline-time:
+
+Edge Server is Offline Longer than Client Maximum Offline Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server uses Device Sync to sync data between the Edge Server instance and
+Atlas. Device Sync has a 
+:ref:`Client Maximum Offline Time <client-maximum-offline-time>` setting.
+
+This setting is enabled by default. It trims history to reduce the amount of
+storage required in the backing Atlas cluster. The longer the client maximum
+offline time, the more storage Atlas requires to keep historical records that
+support Sync.
+
+If an Edge Server instance is offline longer than this setting, this is an 
+unrecoverable error. You must wipe and restart the instance. Alternately, when 
+you bring the instance back online, it will be wiped and restarted 
+automatically.
+
+**This process erases all local unsynced data on the Edge Server instance.**
```

Comment:
the bold formatting here makes it look like it's a new sub-heading, rather than an emphasized line. might be better suited as an admonition?